GitHubCore.ps1

```diff
@@ -735,8 +734,10 @@ function Split-GitHubUri
         repositoryName = [String]::Empty
     }
 
-    if (($Uri -match '^https?://(?:www.)?github.com/([^/]+)/?([^/]+)?(?:/.*)?$') -or
-        ($Uri -match '^https?://api.github.com/repos/([^/]+)/?([^/]+)?(?:/.*)?$'))
+    $hostName = $(Get-GitHubConfiguration -Name "ApiHostName")
```

Comment:
It's fine to keep this as is, but the preceeding `$` is unnecessary here since we're not within a double-quoted string.  Technically the parenthesis are unnecessary too, but are totally acceptable for clarity in expression grouping.  Just calling this out for your awareness.

Reply:
Yep, I waffled a bit on it in light of your code readability feedback and for giggles have spent the last couple of days keeping sub-commands inside nested parenthesis like this to see how the effect grows with more code around it. I'm still undecided on keeping parenthesis or leaving them off so I'll defer to your choice since you're dealing with this code more often than I.